### PR TITLE
Paper style and grammar edits

### PR DESCRIPTION
**Pull request by philipplk**

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ authors:
   - name: Julian Karl Bauer^[corresponding author]
     orcid: 0000-0002-4931-5869
     affiliation: "1" # (Multiple affiliations must be quoted)
-#  - name: Author Without ORCID^[co-first author] # note this makes a footnote saying 'co-first author'
-#    affiliation: 2
+  - name: Philipp Lothar Kinon^[co-first author] # note this makes a footnote saying 'co-first author', feel free to hand over this item to another auther
```
Comment (JulianKarlBauer) [inline paper/paper.md:15]:
```suggestion
  - name: Philipp Lothar Kinon^[co-first author] # note this makes a footnote saying 'co-first author', feel free to hand over this item to another author
```

File: paper/paper.md [hunk 2]
```diff
@@ -32,25 +33,24 @@ bibliography: paper.bib
...
-As linear mappings between the observed deformations and possibly causal stresses,
-fourth level tensors have a special task in the field of linear elasticity.
-Tensors of second- and fourth-order tensors are the primary field of application
-for the methods in `mechkit` and are motivated by research in the areas of
+that simplify common tasks - especially in the area of tensor algebra and tensor notation.
+
+In the context of engineering applications accounting for three spatial dimensions, deformations and stresses in solids are
+usually described by second order tensors.
+As linear mappings between observed deformations and possibly causal stresses,
+fourth order tensors have a special task in the field of linear elasticity.
```
Comment (JulianKarlBauer) [inline paper/paper.md:41]:
```suggestion
fourth-order tensors have a special task in the field of linear elasticity.
```

File: paper/paper.md [hunk 3]
```diff
@@ -32,25 +33,24 @@ bibliography: paper.bib
...
-Deformations and stresses in solids in our three dimensional world are
-usually described in the context of engineering applications by second order tensors.
-As linear mappings between the observed deformations and possibly causal stresses,
-fourth level tensors have a special task in the field of linear elasticity.
-Tensors of second- and fourth-order tensors are the primary field of application
-for the methods in `mechkit` and are motivated by research in the areas of
+that simplify common tasks - especially in the area of tensor algebra and tensor notation.
+
+In the context of engineering applications accounting for three spatial dimensions, deformations and stresses in solids are
+usually described by second order tensors.
```
Comment (JulianKarlBauer) [inline paper/paper.md:39]:
```suggestion
usually described by second-order tensors.
```

File: paper/paper.md [hunk 4]
```diff
@@ -32,25 +33,24 @@ bibliography: paper.bib
...
-
-Deformations and stresses in solids in our three dimensional world are
-usually described in the context of engineering applications by second order tensors.
-As linear mappings between the observed deformations and possibly causal stresses,
-fourth level tensors have a special task in the field of linear elasticity.
-Tensors of second- and fourth-order tensors are the primary field of application
-for the methods in `mechkit` and are motivated by research in the areas of
+that simplify common tasks - especially in the area of tensor algebra and tensor notation.
+
+In the context of engineering applications accounting for three spatial dimensions, deformations and stresses in solids are
```
Comment (JulianKarlBauer) [inline paper/paper.md:38]:
```suggestion
In the context of engineering applications within three spatial dimensions, deformations and stresses in solids are
```

File: paper/paper.md [hunk 5]
```diff
@@ -32,25 +33,24 @@ bibliography: paper.bib
...
+usually described by second order tensors.
+As linear mappings between observed deformations and possibly causal stresses,
+fourth order tensors have a special task in the field of linear elasticity.
+To this end, the methods in `mechkit` are primarily focussed on second and fourth order tensors. Main motivations can thus be found in the research in
```
Comment (JulianKarlBauer) [inline paper/paper.md:42]:
```suggestion
To this end, the methods in `mechkit` are primarily focussed on second- and fourth-order tensors. Main motivations can thus be found in the research in
```

File: paper/paper.md [hunk 6]
```diff
@@ -32,25 +33,24 @@ bibliography: paper.bib
...
 [@Cowin1992], [@Rychlewski2000], [@Spencer1970], [@Boehlke2001], [@Brannon2018]
 and the description of microstructures of fiber reinforced
 composite materials
 [@Bauer2021], [@Kanatani1984], [@Advani1987].
 
-The implementations aim at simplicity in use as well as understandable source code
-and do not put primary emphasis on performance.
-Furthermore the implementations follow, as directly as possible,
-the notation and formulation of the formulas in the scientific sources.
+The implementations in `mechkit` aim at usability and seek to provide understandable source code
```
Comment (JulianKarlBauer) [inline paper/paper.md:50]:
```suggestion
The implementations in `mechkit` aim at usability, seek to provide understandable source code
```

Comment (JulianKarlBauer) [reply paper/paper.md:50]: I like to stick to one "and" in each sentence

File: paper/paper.md [hunk 7]
```diff
@@ -60,16 +60,20 @@ sources is strived for.
...
 the reliability of the implementations.
 
 The main goal of the project `mechkit` is the reuse of research code,
 to increase the reliability of the research results and
 accelerate and simplify further research.
+<<<<<<< HEAD
+`Mechkit` is inspired by [@fiberoripy] and the projects of the author of [@meshio].
+=======
 Mechkit is inspired by [@fiberoripy] and the projects of the author of [@meshio]
+>>>>>>> 8cbb37baff279207d0de895657e95324a94c149a
```
Comment (JulianKarlBauer) [inline paper/paper.md:76]:
```suggestion
`Mechkit` is inspired by [@fiberoripy] and the projects of the author of [@meshio].
```

Comment (JulianKarlBauer) [reply paper/paper.md:76]: I think some merge artefacts found there way in here...

File: paper/paper.md [hunk 8]
```diff
@@ -78,28 +82,27 @@ and is used, besides numerous closed-source projects, in
 In the overlapping area of theoretical continuum mechanics, experimental
 material modeling
 and the numerical solution methods for boundary value problems
-a multitude of different notations exists.
+a multitude of different notations exist.
 As an example, consider the description of the mechanical properties of a
-homogeneous and isotropic, i.e. direction-independent, material within the framework of the
-simplest theory, linear elasticity.
+homogeneous and isotropic, i.e. direction-independent, material within the framework of linear elasticity.
```
Comment (JulianKarlBauer) [inline paper/paper.md:87]: perfect

File: paper/paper.md [hunk 9]
```diff
@@ -78,28 +82,27 @@ and is used, besides numerous closed-source projects, in
...
 Such a material can be described identically by two scalar material parameters.
 However, in the disciplines mentioned above, at least six different material parameters are commonly used
 and motivated by different applications and measurement methods.
 This results in fifteen possible combinations of scalar descriptions of an
-isotropic material, which can be combined to a fourth-order tensor.
-For the resulting tensor, again different notations exist following
-Voigt, Kelvin-Mandel as well as interfaces of open and commercial finite element codes.
+isotropic material, which can be combined to a fourth order elasticity tensor.
```
Comment (JulianKarlBauer) [inline paper/paper.md:92]:
```suggestion
isotropic material, which can be combined to a fourth-order elasticity tensor.
```

File: paper/paper.md [hunk 10]
```diff
@@ -78,28 +82,27 @@ and is used, besides numerous closed-source projects, in
...
+For this tensor, again, different notations exist. They either follow the
+Voigt or Kelvin-Mandel notation or take account of the interfaces between open and commercial finite element codes.
 
-Mechkit" allows an easy exchange between the mentioned notations with user-friendly
+`Mechkit` allows an easy exchange between the abovementioned notations with user-friendly
```
Comment (JulianKarlBauer) [inline paper/paper.md:96]: Nice, the word "abovementioned" is new to me :D

File: paper/paper.md [hunk 11]
```diff
@@ -78,28 +82,27 @@ and is used, besides numerous closed-source projects, in
...
 interfaces and thus helps to avoid errors.
 Since the necessary number of translation functions between different notations
 increases drastically with increasing number of notations,
-translation between all notation might not be practical.
-For the case of notations of second- and fourth-order tensors,
-the shortest path between source and target notation is determined
-in the graph of supported notations, see \autoref{fig:stiffness_graph}.
+translation between all notations might not be practical.
+For the case of notations of second and fourth order tensors,
```
Comment (JulianKarlBauer) [inline paper/paper.md:101]:
```suggestion
For the case of notations of second- and fourth-order tensors,
```

File: paper/paper.md [hunk 12]
```diff
@@ -78,28 +82,27 @@ and is used, besides numerous closed-source projects, in
...
+the shortest path between source and target notation is determined by
+the graph of supported notations, see \autoref{fig:stiffness_graph}.
 
-![Currently supported notations of fourth-order stiffness tensors.\label{fig:stiffness_graph}](stiffness_graph.png){ width=60% }
+![Currently supported notations of fourth order stiffness tensors.\label{fig:stiffness_graph}](stiffness_graph.png){ width=60% }
```
Comment (JulianKarlBauer) [inline paper/paper.md:105]:
```suggestion
![Currently supported notations of fourth-order stiffness tensors.\label{fig:stiffness_graph}](stiffness_graph.png){ width=60% }
```

**Review by JulianKarlBauer — CHANGES_REQUESTED** [review]
Hey,
thank you for the review of the paper draft! That's really valuable.
Left you some small comments.


To the best of my knowledge, it is more common to use 
"second-order tensor" to refer to tensors of second order.

Is this right? If you are fine with it, just use the comments below.

**Comment by philipplk** [comment]
I accepted your proposals and merged.

**Pull request merged**